ara/ui/templates/host_index.html

```diff
@@ -26,6 +26,18 @@
                 <input type="text" class="form-control" id="name" name="name" placeholder="ex: localhost (or) hostname.fqdn" value="{{ search_form.name.value | default_if_none:'' }}">
```

Comment:
Should we disambiguate ``name`` here and use ``host_name`` instead ?

Reply:
do you mean the label or the the appearances in the code?

Reply:
Both, I think ? It becomes useful because we now have two names, the host name and the playbook name (if there is one).

For the label it could be "Host name" and the variable host_name ?

Reply:
Oh, I remembered about this when implementing the task filter and I think I see what you mean.

I think the filter for the host's name should remain "name" but the label and query argument be host_name.
Does that make sense ?

Reply:
Yeah nevermind, let's do just the label, it's simpler that way :p